src/components/exploreCard.jsx

```diff
@@ -165,4 +167,4 @@ const ExploreCard = ({ dao }) => {
   );
 };
 
-export default ExploreCard;
+export default withRouter(ExploreCard);
```

Comment:
This isn't really a pattern we use in this app. You can access history.push by using react-router's useHistory hook instead.